src/components/mobile/mobile-next-query-preview.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <NextQueryPreview
+    #default="{ results, totalResults, suggestion }"
+    class="
+      x-list x-list--gap-05
+      x-background--neutral-95
+      x-padding--06 x-padding--bottom-06
+      x-margin--top-10 x-margin--bottom-05
+    "
+    :suggestion="nextQuery"
+  >
+    <span class="x-title2 x-font-weight--bold">{{ $t('nextQueryPreview.title') }}</span>
+    <div class="x-list x-list--horizontal x-list--align-center x-list--justify-space-between">
+      <div class="x-list x-list--horizontal x-list--align-center x-list--gap-02">
```

Comment:
It is just 1 or 2px effect, but these items should be aligned to the baseline.

```suggestion
      <div class="x-list x-list--horizontal x-list--align-baseline x-list--gap-02">
```